PublicLands.tm2/style.mss

```diff
@@ -1,25 +1,130 @@
 Map { font-directory: url("./fonts"); }
 
-#lands[adminLevel='federal'] {
-    polygon-fill:red;
-}
-#lands[adminLevel='state'] {
-    polygon-fill:green;
-}
-#lands[adminLevel='county'] {
-    polygon-fill:yellow;
-}
-#lands[adminLevel='local'] {
-    polygon-fill:blue;
-}
-
 #lands {
-    text-face-name: 'Source Sans Pro Regular';
-    text-name:'[adminAgency]';
+	text-face-name: 'Noto Sans Regular';
```

Comment:
These labels need a min zoom, maybe 14? The text should be offset so it's on the inside of the polygon, instead of on the line.